k8s-cluster/base/raptor/apps_v1_deployment_raptor-admin.yaml

```diff
@@ -5,7 +5,6 @@ metadata:
     app.kubernetes.io/component: raptor-admin
   name: raptor-admin
 spec:
-  replicas: 1
```

Comment:
not sure why this is removed

Reply:
Replicas defaults to 1 anyway https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#replicas

Reply:
Default is 1, so it's fine.